packages/venia-concept/src/components/PurchaseDetailsPage/PurchaseDetails/selectors.js

```diff
@@ -24,4 +24,62 @@ export const getOrderSummary = () => [
     { property: 'Estimated Tax', value: '$7.50' }
 ];
 
-export const getOtherItems = () => [{}, {}];
+export const getItem = () => {
+    return {
+        id: 1,
+        title: 'Pretty Shorts',
+        size: '43',
+        color: 'Navy Blue',
+        qty: 1,
+        titleImageSrc: 'dress.png',
```

Comment:
Could you please specify here some url, you can get it from magento product page, just use it here.